aircan/dags/api_ckan_import_to_bq_v2.py

```
@@ -0,0 +1,103 @@
+
+    # sample bq_table_id: "bigquerytest-271707.nhs_test.dag_test"
+    bq_table_id = '%s.%s.%s' % (bq_project_id, bq_dataset_id, bq_table_name)          
+    logging.info('Importing %s to BQ %s' % (gc_file_url, bq_table_id))
+    ckan_conf = context['params'].get('ckan_config', {})
+    ckan_conf['resource_id'] = context['params'].get('resource', {}).get('ckan_resource_id')
+    dag_run_id = context['dag_run'].run_id
+    res_id = ckan_conf.get('resource_id')
+    try:
+        bq_import_csv(bq_table_id, gc_file_url, schema, ckan_conf)
+        status_dict = {
+        'dag_run_id': dag_run_id,
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)